mobile/src/main/java/org/openhab/habdroid/background/tiles/AbstractTileService.kt

```diff
@@ -77,14 +107,43 @@ abstract class AbstractTileService : TileService() {
         tile.apply {
             state = Tile.STATE_INACTIVE
             label = data?.tileLabel ?: getString(R.string.tile_number, ID)
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
-                subtitle = getString(R.string.app_name)
-            }
             icon = Icon.createWithResource(this@AbstractTileService, getIconRes(applicationContext, data?.icon))
             updateTile()
         }
     }
 
+    @RequiresApi(Build.VERSION_CODES.Q)
+    private fun updateTileSubtitle() {
+        val lastInfo = WorkManager
+            .getInstance(applicationContext)
+            .getWorkInfosByTag(BackgroundTasksManager.WORKER_TAG_PREFIX_TILE_ID + ID)
+            .get()
+            .lastOrNull()
+        var lastWorkInfoState = lastInfo?.state
+        val timestamp = lastInfo?.outputData?.getLong(ItemUpdateWorker.OUTPUT_DATA_TIMESTAMP, 0) ?: 0
+        if (lastWorkInfoState?.isFinished == true && timestamp < System.currentTimeMillis() - 5 * 1000) {
+            lastWorkInfoState = null
+        }
+        @StringRes val statusRes = when (lastWorkInfoState) {
+            WorkInfo.State.ENQUEUED, WorkInfo.State.BLOCKED -> R.string.item_update_short_status_waiting
+            WorkInfo.State.RUNNING -> R.string.item_update_short_status_sending
+            WorkInfo.State.FAILED, WorkInfo.State.CANCELLED -> R.string.item_update_short_status_failed
+            WorkInfo.State.SUCCEEDED -> R.string.item_update_short_status_succeeded
+            null -> R.string.empty_string
+        }
+        qsTile?.apply {
+            subtitle = getString(statusRes)
+            updateTile()
+        }
+        subtitleUpdateJob?.cancel()
+        if (statusRes != R.string.empty_string) {
```

Comment:
```suggestion
        if (statusRes == R.string.item_update_short_status_succeeded || statusRes == R.string.item_update_short_status_failed) {
```
All other events are triggered by the work info listener. In an error case there's still the error notification.